src/source-pcap.c

```diff
@@ -62,6 +80,7 @@ typedef struct PcapThreadVars_
     const char *bpf_filter;
 
     time_t last_stats_dump;
+    PcapStats64 last_stats64;
```

Comment:
There could probably be a better location cache-line wise. Maybe closer to "ThreadVars *tv" as they are used in the same code?